schema-source/ASHRAE229_extra.schema.yaml

```diff
@@ -1715,7 +1715,7 @@ Subsurface:
       AppG Used By TCDs: True
       AppG P_RMD Equals U_RMD: True
       AppG B_RMD Equals P_RMD: False
-      AppG B_RMDs Same: "unknown"
+      AppG B_RMDs Same: True
```

Comment:
I think that the space conditioning category could change for the baseline and different thermal properties be modeled across rotations. This is not explicitly addressed in 90.1 2019 but is in 90.1 2022. Maybe this should be false then?